Source/AwsCommonRuntimeKit/mqtt/MqttRequestResponseClient.swift

```diff
@@ -358,30 +509,32 @@ internal class MqttRequestResponseClientCore: @unchecked Sendable {
     }
   }
 
-  /// create a stream operation, throws CRTError if the creation failed. You would need call open() on the operation to start the stream
-  public func createStream(streamOptions: StreamingOperationOptions) throws -> StreamingOperation {
-    // TODO: create streamming operation
-    return StreamingOperation()
+  /// Creates a stream operation, throws CRTError if the creation failed. You will need to call open() on the operation to start the stream
+  fileprivate func createStream(streamOptions: StreamingOperationOptions) throws
+    -> StreamingOperation
+  {
+    return try StreamingOperation(streamOptions: streamOptions, client: self)
   }
 
-  /// release the request response client. You must not use the client after call `close()`.
-  public func close() {
+  /// Releases the request response client. You must not use the client after calling `close()`.
+  fileprivate func close() {
     aws_mqtt_request_response_client_release(self.rawValue)
     self.rawValue = nil
   }
 
 }
 
+/// The MqttRequestResponseClient is a client that allows you to send requests and receive responses over MQTT.
 public class MqttRequestResponseClient {
   fileprivate var clientCore: MqttRequestResponseClientCore
 
   /// Creates a new request-response client using an MQTT5 client for protocol transport
   ///
   /// - Parameters:
-  ///     - mqtt5Client: protocolClient MQTT client to use for transport
-  ///     - options: request-response client configuration options
+  ///     - mqtt5Client: the MQTT5 client that use for protocol transport
```

Comment:
Trivial: "MQTT5 client that the request-response client should use as transport"